server/protocols/jmap-rfc-8621/src/test/scala/org/apache/james/jmap/rfc/RequestObjectTest.scala

```diff
@@ -0,0 +1,204 @@
+/** **************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ * *
+ * http://www.apache.org/licenses/LICENSE-2.0                 *
+ * *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ * ***************************************************************/
+
+package org.apache.james.jmap.rfc
+
+import org.apache.james.jmap.rfc.model.CreatedIds.{ClientId, Id, ServerId}
+import eu.timepit.refined.auto._
+import eu.timepit.refined.api.Refined
+import org.apache.james.jmap.rfc.model.CreatedIds.{ClientId, ID, ServerId}
+import org.apache.james.jmap.rfc.model.Invocation.{Arguments, MethodCallId, MethodName}
+import org.apache.james.jmap.rfc.model.RequestObject.Capability
+import org.apache.james.jmap.rfc.model.{CreatedIds, Invocation, RequestObject}
+import org.scalatestplus.play.PlaySpec
```

Comment:
I dont' get why you need the library `org.scalatestplus.play.PlaySpec` and to extends `PlaySpec`.
Why did you chose to use it ?

Reply:
When i learn scala stuffs, i found it for testing scala and play-json.
Why i chose it: because is easy to write and follow, the structure is more similar with we are talking.